games/s.yaml

```diff
@@ -401,6 +401,22 @@
   type: remake
   updated: 2018-06-15
 
+  name: 'Trust & Betrayal: The Legacy of Siboot'
```

Comment:
```suggestion
- name: 'Trust & Betrayal: The Legacy of Siboot'
```

Reply:
I'm sorry, I don't get the change.

Reply:
this is yaml syntax for creating a new array item; without it this would be part of the previous item

Reply:
Ok! gotcha!